Cargo.toml

```diff
@@ -48,3 +48,14 @@ resolver = "2"
 [patch."https://github.com/purestake/moonbeam"]
 pallet-author-mapping = { git = "https://github.com/zeitgeistpm/moonbeam", branch = "downgrade-staking-and-mapping" }
 parachain-staking = { git = "https://github.com/zeitgeistpm/moonbeam", branch = "downgrade-staking-and-mapping" }
+
```

Comment:
Can we suppress compiling debug symbols for some of the heavy crates that are very unlikely to be the cause of an error, like [Paritytech](https://github.com/paritytech/polkadot/blob/09f340c928ea324105b39b957dc155cbd29b173d/Cargo.toml#L117-L171) did?

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/789/commits/9907667a5bf0e5ddadb1af4e91cb2bd37db853e8